### PR TITLE
Custom login form

### PR DESCRIPTION
**Pull request by edifirut**

### CODE REVIEW

File: packages/apps/contacts/components/organisms/login-panel/ui/NodeText.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { UiNode, UiNodeTextAttributes } from "@ory/client"
+import { UiText } from "@ory/client"
+import { CodeBox, P } from "@ory/themes"
+import styled from "styled-components"
```
Comment (Kasia9090) [inline packages/apps/contacts/components/organisms/login-panel/ui/NodeText.tsx:4]: We don't use styled components, let's stick with one approach

Comment (Kasia9090) [reply packages/apps/contacts/components/organisms/login-panel/ui/NodeText.tsx:4]: it can stay for now but it should be removed later

**Review by Kasia9090 — APPROVED** [review]
Note: we should drop styled-components when we move this code to the new repo

File: packages/apps/contacts/components/organisms/login-panel/ui/NodeText.tsx (hunk 1 above)
Comment (edifirut) [reply packages/apps/contacts/components/organisms/login-panel/ui/NodeText.tsx:4]: we are going to clean this up. we need to understand how ory is giving us the nodes back in order to do our own layout and design around them but

**Pull request merged**